web/viewer.css

```diff
@@ -915,15 +914,15 @@ select {
 a.secondaryToolbarButton {
   padding-top: 6px;
   text-decoration: none;
+  display: inline-block;
```

Comment:
Given that the existing code should be working, I don't understand what this change is actually for?

Reply:
The highlight was not correct when hovering the menu item:
<img width="228" alt="image" src="https://user-images.githubusercontent.com/5641725/188275578-db083c94-0ffb-44fb-8ee3-44e7b07c7c6f.png">
If there is a better fix, please tell me.

Reply:
This might be a perfectly fine change, but I'd like to understand *why* it's necessary since it wasn't in the old code (when the button became visible at a small viewer-width).

Could it possibly have been that the following rule "fixed" that, and thus obscured that this was always broken?
https://github.com/mozilla/pdf.js/blob/50d72fc111683ac3ee39a7678c9dd6afe94f08e6/web/viewer.css#L1506-L1508

Reply:
The default display value for a `<button>` is `inline-block` and for `inline` for a `<a>`.
When we're in the state you mentioned, `inherit` will give the display value of the containing `div` which is by default `block`.

My feeling is that it's a bit messy and we should likely be more explicit in specifying that all the direct children of the secondary toolbar should be `block` or `inline-block`, ... something like that.
Anyway, I'll remove this "fix" from this patch because it's unrelated and we should fix that stuff in an other patch.